### DISCUSSION TITLE
Can ngrinder test the webservice interface

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#824] [Q&A]
the url is end by "wsdl",and The parameter is an XML file

**Comment by donggyu04** [comment]
@zhougy1989 

Check the grinder's documents ngrinder base on it. http://grinder.sourceforge.net/g3/script-gallery.html
and you can add your client class or 3rd party library on classpath(add jar to lib folder) and freely  import them in script.
